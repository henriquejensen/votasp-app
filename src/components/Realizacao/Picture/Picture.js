import React from 'react';
import './Picture.css';

const Picture = (props) => {
  return (
    <section className='Picture'>
      <img src={props.picture} alt={props.altPicture} />
      <span>{props.name}</span> <br />
      <span>{props.role}</span>
    </section>
  );
}

export default Picture;
