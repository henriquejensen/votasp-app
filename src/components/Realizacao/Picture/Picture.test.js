import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Gabriel from '../../img/realizacao/GRibeiro.png';

import Picture from './Picture';

describe('Picture', () => {
  it('should match a snapshot', () => {
    const person = { name: 'Gabriel Ribeiro', picture: Gabriel, role: 'Developer', altPicture: 'Foto da realizadora' }
    expect(toJson(shallow(
      ( <Picture
          name={person.name}
          picture={person.picture}
          role={person.role}
        /> 
      )
    ))).toMatchSnapshot();
  })
})
