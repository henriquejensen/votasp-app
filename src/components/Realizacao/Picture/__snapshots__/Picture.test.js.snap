// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture should match a snapshot 1`] = `
<section
  className="Picture"
>
  <img
    src="test-file-stub"
  />
  <span>
    Gabriel Ribeiro
  </span>
   
  <br />
  <span>
    Developer
  </span>
</section>
`;
